xwiki-platform-distribution/xwiki-platform-distribution-war-legacydependencies/pom.xml

```diff
@@ -86,6 +86,10 @@
       <type>pom</type>
```

Comment:
You should also add `xwiki-commons-cache-api` to the banned dependencies, above (makes easier to not miss any exclusion).

Reply:
Good point, thanks!
Done here https://github.com/xwiki/xwiki-platform/pull/3573/files#diff-eb46a6f118b4a9986749e21902858a951bd0f9b1a8e9f513e4d488d3c81acbb6R57